.travis.yml

```diff
@@ -1,3 +1,10 @@
+dist: trusty
+sudo: required
+services:
+  - docker
+git:
+  depth: 2
```

Comment:
I looked up what this means and found:

https://docs.travis-ci.com/user/customizing-the-build#Git-Clone-Depth

but I don't understand the reason for going with depth 2 rather than 5 or some other value.